packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -565,6 +570,7 @@ class AnswerOption extends React.Component<
 
     handleImSure = () => {
         this.props.onDelete();
+        this.handleCancelDelete();
```

Comment:
Ooo, what's this do and why is it here?

Reply:
Its an existing function that closes the "I'm sure" button, I added it so it wouldn't stay open once someone deletes the current item

Reply:
Here is the before: 
https://github.com/user-attachments/assets/8838f716-dfd4-4aa7-995d-b15f4c5aa22a

Here is the after: 

https://github.com/user-attachments/assets/8766dcda-036d-4298-906f-018aec5b0859